docs/core-library/h3indexing.md

```diff
@@ -34,10 +34,12 @@ The components of the **H3** unidirectional edge index (mode 2) are packed into
 * 7 bits to indicate the base cell 0-121, and
 * 3 bits to indicate each subsequent digit 0-6 from resolution 1 up to the resolution of the cell (45 bits total is reserved for resolutions 1-15)
 
-The canonical string representation of an **H3Index** is the hexadecimal representation of the integer.
+The canonical string representation of an **H3Index** is the hexadecimal representation of the integer, using lowercase letters.
 
 The three bits for each unused digit are set to 7.
 
+The highest order bit of an **H3Index** must be set to 0.
```

Comment:
Worth adding something about how this ensures a positive integer?

Reply:
Unless it's considered a `uint64` where it'd be positive no matter what. :)

I think we did it just to reduce the amount of crap we'd have to deal with in Java.

Reply:
This feels weird to have stated at the end here. What about rewording the "the lowest order 63 bits of a 64-bit integer" stuff, and add "The highest order bit of an **H3Index** must be set to 0" to the beginning of the list above? That would keep everything in order.

Reply:
@ajfriend good suggestion, better to rework that text to avoid ambiguity about only using some of the bits.